code/controllers/subsystem/vote.dm

```diff
@@ -12,23 +13,10 @@ var/global/list/round_voters = list() //Keeps track of the individuals voting fo
 	var/list/voted = list()
 	var/list/voting = list()
 	var/list/current_votes = list()
+	var/list/round_voters = list()
 	var/auto_muted = 0
 
-/datum/controller/vote/New()
-	if(vote != src)
-		if(istype(vote))
-			qdel(vote)
-		vote = src
-	spawn(0)
-		while(!QDELETED(src))
-			try
-				while(!QDELETED(src))

```

Comment:
disgusting.

Reply:
![image](https://user-images.githubusercontent.com/25063394/56475014-5c5e9180-647a-11e9-8bc4-6006fd4ec5fb.png)
In all seriousness what do you want me to change it do